### DISCUSSION TITLE
The time in the Timestamp tag is not the same as the system time.

### DISCUSSION THREAD

**Discussion by feribudi2** [frangoteam/FUXA#1338] [General]
Dear Admin, I tried to publish mqtt data but the time in the timestamp is not the same as the time in the system. On the FUXA navigation bar header, the time shows that it is correct and the same as the system, but the tag value timestamp is not the same, how do I match this?
<img width="583" alt="timestamp format" src="https://github.com/user-attachments/assets/d696391f-988b-4849-8fa7-1a6606831efd">

This is my fuxa time
<img width="164" alt="time fuxa ok" src="https://github.com/user-attachments/assets/1331ed27-fda0-4b63-97a6-f3234f9cd177">

**Comment by unocelli** [comment]
Hi, I suppose that is the time of a tag, in FUXA yet we don't have a system timestamp to bind
